### DISCUSSION TITLE
How to escape Storybook MDX Meta title?

### DISCUSSION THREAD

**Discussion by coreybruyere** [storybookjs/storybook#12374] [Help]
I have a monorepo. I want to organize each story in storybook beneath each components parent package. Inside of stories I'm looking to do something like this:

```
<Meta title="Packages/@organization/package/Box" component={Box} />
```

How can I escape the slash between `@organization` and `package` so that both org and package are grouped together as a heading in the sidebar like so:

![Screen Shot 2020-09-03 at 12 05 27 PM](https://user-images.githubusercontent.com/3430222/92156431-d11b3d00-eddd-11ea-9330-3e7c9b703436.png)

**Comment by shilman** [comment]
This isn't currently supported. I think we discussed escaping via backslashes, but never implemented:

```
<Meta title="Packages/@organization\\/package/Box" component={Box} />
```

@tmeasday WDYT?

**Comment by tmeasday** [comment]
Can you use a different character, perhaps one that looks like a `/` but isn't? I think we discussed the escaping but it seemed unnecessarily complex.
